schema.graphql

```diff
@@ -249,6 +249,25 @@ input SavedItemUpsertInput {
   """
   timestamp: Int
 }
+
+"""
+Input field for upserting a Save. Used by saveUpsert mutation
+"""
+input SaveUpsertInput {
+  """
+  The url to create/update the SavedItem with. (the url to save to the list)
+  """
+  url: String!
+  """
+  Optional, create/update the SavedItem as a favorited item
+  """
+  isFavorite: Boolean
+  """
+  Optional, time that request was submitted by client epoch/unix time
+  """
+  timestamp: Int
```

Comment:
ok we don't need this timestamp as per the schema proposal:
```
input SaveUpsertInput {
    url: String!
    title: String
    isFavorite: Boolean
  }
```
https://getpocket.atlassian.net/wiki/spaces/PE/pages/2832760905/Accepted+Proposal+Consolidation+V1+Pocket-Graph#Bulk-Mutations